backend/schemas/structures/wmp_params.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class WatchPixelParams(BaseModel):
+    lat: float
+    lon: float
+    date: datetime
+    path: int
```

Comment:
Try using our existing types, that for example implement checks for maximum values